website/docs/install.md

```diff
@@ -89,7 +89,7 @@ You can alter the variables in `charts/gatekeeper/values.yaml` to customize your
 If you used a prebuilt image to deploy Gatekeeper, then you can delete all the Gatekeeper components with the following command:
 
   ```sh
-  kubectl delete -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/release-3.5/deploy/gatekeeper.yaml
+  kubectl delete -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/deploy/gatekeeper.yaml
```

Comment:
this one is a little tricky as this will be copied to be v3.8 (next release), we need a way to substitute these versions when we do a release. related #1688

Reply:
I thought so too. Tried to keep with consistent with the what is there with install. And the automation could replace "master" keyword with the release info